### ISSUE TITLE
Bom'bay <Hexendoktor in Ausbildung> in Sen'Jin

### ISSUE THREAD

**Issue by laedron** [ClassicWoW/Nefarian_1.12.1_Bugtracker#1567]
**Welches Verhalten wird beobachtet?**
Nachdem man die Quest Zalzane erledigt hat bietet Bom'Bay eine Anzahl beliebiger Verzauberungen an. soweit so gut.
--->>>>> Klickt man aber auf diese passiert gar nichts.

**Wie sollte es sich verhalten?**
After completing the quest Horde 15 [10] Zalazane, he can provide any of number of random hexes, which most often get you turned into a critter (you can dispel this by right mouse clicking the buff). Sometimes his hexes produce cool effects such as being drunk, ghostly, getting thrown into the air (make sure you have a full health bar for this as the "feather" effect, when you fall back down, can trigger too late on rare occasions and you will fall to your death), or becoming enlarged for 5 minutes (very funny when you add other effects such as bloodlust and a "bug" from the gnomish engineering trinket Gnomish Poultryizer). Strangely enough, he is one of the only, if not the only, Trolls in World of Warcraft with his speech set. 

**Schritte zur Reproduzierung**
Quatscht ihn an.

**Zusätzliche Informationen (Screenshots, Videos, etc.)**
![bom bay in sen jin](https://cloud.githubusercontent.com/assets/19352170/15268724/0381f478-19e9-11e6-8147-a9fdd2edb118.png)

**Datenbank Link**
http://datenbank.classic-wow.org/?npc=10578

**Comment by Slimmer2** [comment]
Quelle?

**Comment by laedron** [comment]
http://wowwiki.wikia.com/wiki/Bom'bay

**Comment by Slimmer2** [comment]
http://wowwiki.wikia.com/wiki/Bom%27bay?oldid=440679
http://datenbank.classic-wow.org/?quest=826

**Comment by Dotox** [comment]
Kann das jemand bestätigen?

**Comment by Badorr** [comment]
Ja, keine der angebotenen Optionen hat irgendeinen Effekt. Der NPC sollte aber, wie in den Quellen beschrieben, den Spieler verhexen sobald man die Quest erledigt hat.
